docs/src/examples/components/Dialog/Content/DialogExampleHeaderAction.shorthand.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import { Button, Dialog } from '@stardust-ui/react'
+import { useBooleanKnob } from '@stardust-ui/docs-components'
+
+const DialogExampleHeaderAction: React.FC = () => {
+  const [open, setOpen] = useBooleanKnob({ name: 'open' })
+  return (
+    <Dialog
+      open={open}
```

Comment:
Not sure that this example is 100% descriptive.

If `headerAction` is used for a close button (I bet most common case) like in this example, the component must be obviously controlled = consumer must also handle `onCancel` and `onConfirm` in order to close the dialog after clicking any action button or outside the dialog.

This example does not do that (to show just the header) but it can be confusing for a consumer.

Reply:
Maybe we should add more examples for the Dialog for controlling state/custom handlers. Do you suggest any change in this particular example? I mean we can remove the open knobs and do nothing when clicking the close icon, but I think that would be even more confusing..

Reply:
Agree. Let's leave it as it is for now.